apps/studio/src/lib/db/sql_tools.ts

```diff
@@ -11,6 +15,64 @@ export function splitQueries(queryText: string, dialect) {
   return result
 }
 
+export function defaultParamTypesFor(dialect: string): ParamTypes {
+  switch (dialect) {
+    case 'psql':
+      return {
+        numbered: ['$'],
+      };
+    case 'mssql':
+      return {
+        named: [':'],
+      };
```

Comment:
Could I suggest adding the `named: [':']` for mysql as well here? 😁
```suggestion
      };
    case 'mysql':
      return {
        positional: true,
        named: [':'],
      };
```